metrics-core/src/test/java/com/codahale/metrics/DerivativeGaugeTest.java

```diff
@@ -1,23 +1,27 @@
 package com.codahale.metrics;
 
+import org.junit.Before;
 import org.junit.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 public class DerivativeGaugeTest {
-    private final Gauge<String> gauge1 = new Gauge<String>() {
```

Comment:
I don't think there is a big improvement in replacing a simple gauge with a mock object. I would avoid mocking unless needed. Could you please revert this change?